safe-global/web-core
# fix: remove unnecessary dependency

## What it solves

Resolves `@gnosis.pm/safe-deployments`

## How this PR fixes it

We already have `@safe-global/safe-deployments` installed but were still using `@gnosis.pm/safe-deployments` in a test and listing it on our licenses page.

## How to test it

- Observe all tests passing.
- Observe no `@gnosis.pm/safe-deployments` presence on the licenses page.

## Checklist
